### ISSUE TITLE
No link to "all events" present when there's a future event scheduled

### ISSUE THREAD

**Issue by kikito** [phoet/on_ruby#170]
Related with #167 - I got the condition in which the "all events" link is not shown wrong.

This is how madridrb.com looks right now:

![screen shot 2015-02-18 at 10 58 04](https://cloud.githubusercontent.com/assets/63131/6245277/2a571546-b75d-11e4-93e8-e262f31b9fc9.png)

As it is, there is no way to navigate to the `/events` page by clicking.

Bonus points: put a link to "see all events" in events/show, too.

**Comment by phoet** [comment]
there is an "all events" link once the usergroup [has one event that is in the past](https://github.com/phoet/on_ruby/blob/master/app/views/home/_events.slim#L19-L26). i don't see much value in having a link to the all-events page if there is only one of them. is there something special that you have in mind?

**Comment by kikito** [comment]
Well the change I just made ( #173 ) gives at least one reason to always display a link to Events#index from Home#index  - that's where the webcal and rss feed links are shown. Users can't subscribe to them without an initial link in the home.

**Comment by phoet** [comment]
fair enough. speaking of past events, any plans for migrating old data?

**Comment by kikito** [comment]
The updated PR is passing the tests as I write this.

> any plans for migrating old data?

That's going to be ... tricky. The old data resides in a couple markdown wikis. We would have to transform them into something we can use automatically - like yaml or json. The wiki syntax is more or less uniform, but still, extracting data from it requires a lot of human inspection and it'll take time.

I could not enlist the help of the community to do the new madrid.rb website, but I'm thinking that maybe I can convince someone to do the markdown-to-yaml transformation for me. This process is ideal for ruby newbies in fact (it can even be split into several parts).

Long story short: nothing has been done on that regard. I hope I can enlist some help on the next meetup (thursday next week).

**Comment by phoet** [comment]
@kikito are the past events found in [the github wiki](https://github.com/madridrb/madridrb.github.io/wiki)? if it's just 8 of them i would just add them via the admin one at a time.

**Comment by kikito** [comment]
That's just one of them ... the other one is https://madridrb.jottit.com/

**Comment by phoet** [comment]
well, that's quite some more :smile_cat:
